app/invitations/controllers.py

```diff
@@ -123,6 +124,60 @@ def send_request(new_user, committee):
         db.session.rollback()
         return Response.RequestError
 
+##
+## @brief      Sends an invitation to a committee-head to close
+##             a charge.
+##
+## @param      committee       The committee to join.
+## @param      new_user        The user to be invited.
+##
+## @return     True if email sent, False if not.
+##
+def send_close_request(user, committee, chargeID):
+    invite = and_(
+        Invitations.user_name == committee.head,
+        Invitations.committee_id == committee.id,
+        Invitations.isInvite == False
+    )
+    
+    invitation = Invitations (
+        user_name= user.id,
+        committee= committee,
+        committee_id = committee.id,
+        charge_id = chargeID,
+        isInvite=False
+    )
+
+    admins = []
+    admins = db.session.query(Users).filter(Users.is_admin == True)
+
+    admin_emails = []
+    for user in admins:
+        admin_emails.append(user.id + "@rit.edu")
+    
+    try:
+        db.session.add(invitation)
+        db.session.commit()
+        email = {}
+        email["title"] = "Close Charge Request"
+        email["sender"]=("SG TigerTracker", "sgnoreply@rit.edu")
+        email["recipients"] = admin_emails
+        email["subtype"] = "related"
+        email["html"] = render_template(
+            'close_charge_request.html',
+            user_name= committee.head,
+            committee_head= committee.head,
+            charge_name= chargeID,
+            time_stamp= time.time(),
+            app_url= app.config['CLIENT_URL'] + str(invitation.id)
```

Comment:
The template is looking for a `request_url` but `app_url` is being passed instead.